### PR TITLE
auth bugfix + warnings + environment in templates

### PR DESCRIPTION
**Pull request by clux**
minor tinty things that came up today

### CODE REVIEW

File: shipcat_cli/src/kubectl.rs [hunk 1]
```diff
@@ -13,11 +12,10 @@ use kube::{
 };
 
 use k8s_openapi::api::authorization::v1::{
-    ResourceRule,
+    //ResourceRule,
```
Comment (janslow) [inline shipcat_cli/src/kubectl.rs]: Can we just delete these?

Comment (clux) [reply shipcat_cli/src/kubectl.rs]: they are used by a commented out function - i was going to ask jack

Comment (clux) [reply shipcat_cli/src/kubectl.rs]: @jackatbancast

Comment (jackatbancast) [reply shipcat_cli/src/kubectl.rs]: Happy for it to be removed, can revert it if we end up needing it

**Pull request merged**